CONTRIBUTING.md

```diff
@@ -323,9 +325,9 @@ For all OS types perform the following:
 #### RHEL 7/8
 
 1. Run `pyinstaller -F ./pkg_bin/conjur`. Once this is run, a `dist` folder will be created with the executable in it.
-1. Once an executable has been created, run `zip conjur-cli-rhel-7 conjur` (`zip conjur-cli-rhel-8 conjur` for RHEL 8 
-   machines) to zip the file.
-1. Sign the `zip` and add it as an asset in the release page.
+1. Once an executable has been created, run `tar cvf conjur-cli-rhel-7.tar.gz conjur` 
+   (`tar cvf conjur-cli-rhel-8.tar.gz conjur` for RHEL 8 machines) to archive the file.
+1. Sign the archive and add it as an asset in the release page.
```

Comment:
"add them both as assets" because we are taking about two tars

Reply:
let's add something about how we need to push the .sigs as well for signature validation

Reply:
when we run `pyinstaller -F ./pkg_bin/conjur` does it create the same executable for RHEL 7 and RHEL 8? if so, why do even have 2 artifacts? if not - how do you create one per platform?

Reply:
updated the doc according to what i understand. please review

Reply:
you need to spin up two different machines for RHEL 7 and RHEL 8 and then run the `pyinstaller -F ./pkg_bin/conjur` on each. Pyinstaller is a tool that, amongst other things, allows us to pack our CLI into a single executable. 

The limitation of Pyinstaller that you need to compile down the exec on the OS that it is intended to run on. For example, an executable compiled on macOS won't run on Windows. For RHEL, it is interesting because I have seen that an executable compiled on RHEL 8 can work on RHEL 7 but **not** vice versa. "So why not just deliver a RHEL 8 exec?" because although it _seems_ to work, this might not always hold in later versions and just to be on the same side and not OS-dependent, we deliver two.

Reply:
TIL